### DISCUSSION TITLE
[v3.3.0] ReactQuery Devtools in production

### DISCUSSION THREAD

**Discussion by changyeamoon** [TanStack/query#1475] [General] [answered]
In the `react-query-devtools` package we were able to import devtools in production through `dist/react-query-devtools.production.min.js` 

Now that the devtools is included in `react-query`. Can we do the same for `react-query/devtools`?

**Comment by tannerlinsley** [comment, marked as answer]
We ship a UMD bundle for that here: https://unpkg.com/browse/react-query@3.3.0/dist/react-query-devtools.production.min.js

**Reply by changyeamoon** [reply]
awesome thank you! For some reason i didnt see this in my dist folder
